### DISCUSSION TITLE
Error when installing from F-Droid

### DISCUSSION THREAD

**Discussion by sryze** [termux/termux-app#2682] [Q&A] [answered]
Error message in English: 
Cannot install due to an unknown error

Screenshot:
![Screenshot_20220328-114517_F-Droid.jpg](https://user-images.githubusercontent.com/4525736/160333979-c6212420-ce8d-4f8b-9ac9-0181deca553c.jpg)

Any idea why it's failing? I haven't had a chance to check F-Droid logs yet, maybe someone knows how to overcome this?

Android 11 / Galaxy A70

**Comment by sk0kanik** [comment, marked as answer]
"If you wish to install from a different source, then you must uninstall any and all existing Termux or its plugin app APKs from your device first, then install all new APKs from the same new source." - https://github.com/termux/termux-app#installation
